charts/controlplane/Chart.yaml

```diff
@@ -2,14 +2,9 @@ apiVersion: v2
 name: controlplane
 description: Deploys all microservices associated with managing and controlling the roclub connectors
 type: application
-
-# This is the chart version. This version number should be incremented each time you make changes
-# to the chart and its templates, including the app version.
-# Versions are expected to follow Semantic Versioning (https://semver.org/)
 version: 2.1.4
-
-# This is the version number of the application being deployed. This version number should be
-# incremented each time you make changes to the application. Versions are not expected to
-# follow Semantic Versioning. They should reflect the version the application is using.
-# It is recommended to use it with quotes.e
-appVersion: "2.1.4"
\ No newline at end of file
+appVersion: "2.1.4"
+dependencies:
```

Comment:
Can this dependecny be limited to connector-status service only instead of all?

Reply:
die dependencies beziehen sich auf andere Helm-Charts (influxdb2), die von der Anwendung benötigt werden, um korrekt zu funktionieren. Erst wenn influxdb2 deployed wurde, dann wird der dns record gesetzt. Das hat noch nichts mit connector-status zu tun